src/main/java/ivorius/reccomplex/block/TileEntityBlockScript.java

```diff
@@ -68,13 +68,14 @@ public NBTTagCompound writeToNBT(NBTTagCompound nbtTagCompound)
     @Override
     public void generate(StructureSpawnContext context, WorldScriptMulti.InstanceData instanceData)
     {
-        script.generate(context, instanceData, pos);
+    	if (spawnTriggerable || (redstoneTriggerable && redstoneTriggered))
+    		script.generate(context, instanceData, pos);
     }
 
     @Override
     public boolean shouldPlaceInWorld(StructureSpawnContext context, WorldScriptMulti.InstanceData instanceData)
     {
-        return false;
+        return !spawnTriggerable || (redstoneTriggerable && !redstoneTriggered);
```

Comment:
Why should scripts that trigger on spawn _and_ redstone input, but are currently powered not spawn?

Reply:
Sorry, now I get it, it's and AND not OR. The latest commit should properly handle cases where the script block is set to spawn AND redstone triggers.